### PR TITLE
Container selector option

### PR DESCRIPTION
**Pull request by Keaws**
Hi, thank you for creating this library.

I have the case when I'd like to render the toast message and position it relative to the element within the iframe, so adding `ct-container` id won't work as `document.getElementById('ct-container')` is executed in the main window.

This change allows passing an option of a container css selector.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "cogo-toast",
 	"description": "Beautiful, Zero Configuration, Toast Messages - Built with React",
-	"version": "3.2.1",
+	"version": "3.3.0",
```
Comment (anmolmahatpurkar) [inline package.json]: Lets not bump the version here. Will do that as I make a new release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ### Features
 
+- v3.3.0
```
Comment (anmolmahatpurkar) [inline CHANGELOG.md]: Lets not bump the version here. Will do that and add the changelog as I make a new release.

File: src/index.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ import './styles/styles.css';
 let ctToastCount = 0;
 
 const cogoToast = (text, options) => {
-	let rootContainer = document.getElementById('ct-container');
+	let rootContainer = document.querySelector(options.containerCssSelector) || document.getElementById('ct-container');
```
Comment (anmolmahatpurkar) [inline src/index.js]: Can we change it to `toastContainerID`, where we could pass the ID instead of a CSS selector

File: .all-contributorsrc [hunk 4]
```diff
@@ -20,6 +20,15 @@
         "design",
         "doc"
       ]
+    },
```
Comment (anmolmahatpurkar) [inline .all-contributorsrc]: Please resolve the conflicts with the latest master

File: src/index.js (hunk 3 above)
Comment (Keaws) [reply src/index.js]: yep, makes sense

**Pull request merged**